xml/FrameworksIndex/netcore-1.0.xml

```diff
@@ -11240,16 +11240,6 @@
       <Member Id="F:System.IO.FileMode.OpenOrCreate" />
       <Member Id="F:System.IO.FileMode.Truncate" />
     </Type>
-    <Type Name="System.IO.FileNotFoundException" Id="T:System.IO.FileNotFoundException">
```

Comment:
I'm not sure why this is removed. According to apisof.net, this type is available in .NET Core 1.0 from the System.Runtime.dll assembly.

Reply:
That is weird. I copied the configuration parameters from the other CI job, and the dll XML changes aren't touching .NET Core.

Reply:
@gewarren I will take a look tomorrow.

Reply:
@gewarren I was not able to reproduce this in my local. @v-alje would you mind opening a bug regarding this issue? Thanks!

Reply:
@huangmin-ms Sure. Bug has been opened: https://dev.azure.com/ceapex/Engineering/_workitems/edit/639800